### ISSUE TITLE
`./make.py worker-instance` failing due to `policy-reasoner` updates

### ISSUE THREAD

**Issue by Lut99** [epi-project/brane#117]
The current version on `main` fails to compile the `brane-chk` image because its attempting to copy the `policy-reasoner` binary after it has been compiled. However, after epi-project/policy-reasoner#33 has been merged, this binary no longer exists; I think it should instead build the `eflint` binary.

I saw that you (@DanielVoogsgerd) mentioned you got to build _and_ run the entire framework, though. So did you already implement a fix for this...?

**Comment by DanielVoogsgerd** [comment]
Yeah, it is working, it should be fairly trivial to just use the eflint binary instead. I am right now using the noop reasoner as it nice during the setup phase though.

What I personally would like the best is to change `make.py` so that we have multiple targets for the available reasoners.

So `./make.py chk-eflint-image` would generate the eflint reasoner and the `./make.py chk-posix-image` would generate the posix reasoner, etc.

**Comment by Lut99** [comment]
Yeah, sure! Being able to choose between reasoners is very useful, I agree.

Did you implement a fix on one of the branches/PRs, though? I wonder why it fails on my machine.

**Comment by DanielVoogsgerd** [comment]
Well kind off, I am pointing them to my fork where I use one of the two patches for no op reasoner, but those are quite botchy

**Comment by Lut99** [comment]
Ahh OK. Well no worries, I can whip up a fix, just didn't want to cause conflicts with yours :)

**Comment by DanielVoogsgerd** [comment]
Ah, that would be nice. Just as a reminder, not all reasoners have the same input arguments as the eflint reasoner one. So [this line](https://github.com/epi-project/brane/blob/main/docker-compose-worker.yml#L32) would have to be different on other reasoners. I think this is where I got stuck; finding a way to give every reasoner what it needs.
